samples/attestation/README.md

```diff
@@ -275,3 +275,12 @@ To use only the SGX remote functionality:
 * On Linux, use `make runsgxremote` instead of `make run`.
 
 * On Windows, use `ninja runsgxremote` instead of `ninja run`.
+
+## Determining Call Path for SGX Quote Generation
+
+Evidence of format `OE_FORMAT_UUID_SGX_ECDSA` can be generated by either taking the in-process call path or taking the out-of-process call path. However, if you wish to specify the call path it takes to generate a quote, here is what you can do:
```

Comment:
This text currently implies it applies to both Windows and Linux, but if I understand the other comments, this text is only true for Linux.